silk/hatch.toml

```diff
@@ -2,3 +2,8 @@
 
 [[envs.default.matrix]]
 python = ["2.7", "3.8"]
+
+[envs.default]
+dependencies = [
+  "freezegun==1.2.2",
```

Comment:
To reviewer: I'm open to other possible ways to set mock the current time for testing purposes, as I saw that there were [dependency concerns in this old PR](https://github.com/DataDog/integrations-core/pull/6692#discussion_r427474587). I just kept is as `freezegun` since that's what the [`datadog-checks-downloader`](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_downloader/hatch.toml#L14) used for time-dependent tests.